client/test/e2e/Menu.test.ts

```diff
@@ -1,20 +1,11 @@
 // src: https://playwright.dev/docs/writing-tests
 
 import { test, expect } from '@playwright/test';
-
-interface TestLink {
-  text: string;
-  url: string;
-}
-const links: TestLink[] = [
-  { text: 'Library', url: '/library' },
-  { text: 'Digital Twins', url: '/digitaltwins' },
-  { text: 'Workbench', url: '/workbench' },
-];
+import links from './Links.ts'; // Extension is required with Playwright import
 
 test.describe('Menu Links from first page (Layout)', () => {
   test.beforeEach(async ({ page }) => {
-    await page.goto('/');
+    await page.goto('./');
```

Comment:
Why is this relative location required in URLs?

Reply:
It is required in order to include the additional `dtaas` link in the URL. with just `/` it would just navigate to `localhost:4000` and ignore the base URL.
I just tested again and confirmed this behavior.